src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -250,6 +256,12 @@ def tearDown(self):
             try:
                 server_utils.stop_server(hosts=self.hostlist_servers)
             finally:
+                try:
+                    #Storage reset
+                    if self.nvme_parameter == "nvme":
+                        server_utils.storage_reset(self.hostlist_servers)
```

Comment:
Exceptions need to be caught and handled here.  This is another instance of [DAOS-3203](https://jira.hpdd.intel.com/browse/DAOS-3203).

Reply:
will add exception block

Reply:
I was trying to get the self.fail on this exception but it always going to raise it as TestSetupFail as it's part of teardown. 
We can either have TestSkipError, TestCancel specific and rest all exception will be converted to TestSetupFail from avocado code. Added comment in [DAOS-3203](https://jira.hpdd.intel.com/browse/DAOS-3203?focusedCommentId=237678&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-237678) 

Reproduced traceback from: /usr/lib/python2.7/site-packages/avocado/core/test.py:628
Traceback (most recent call last):
  File "/home/samirrav/daos/src/tests/ftest/pool/attribute.py", line 113, in tearDown
    super(PoolAttributeTest, self).tearDown()
  File "/home/samirrav/daos/src/tests/ftest/util/apricot/apricot/test.py", line 263, in tearDown
    server_utils.storage_reset(self.hostlist_servers)
  File "/home/samirrav/daos/src/tests/ftest/util/server_utils.py", line 359, in storage_reset
    raise ServerFailed("Error resetting NVMe storage")
ServerFailed: Error resetting NVMe storage

ERROR 1-./pool/attribute.py:PoolAttributeTest.test_pool_attributes;createmode-createset-createsize-validlongname-validvalue-hosts-server-505d -> TestSetupFail: Error resetting NVMe storage

Reply:
ACK.  I'm investigating with the Avocado team about how this should be handled.